workflow-trigger

```diff
@@ -1 +1 @@
-2022-08-28T20:00:00Z
\ No newline at end of file
+2022-09-02T20:00:00Z
```

Comment:
Hi @123MwanjeMike . Could we avoid this approach and setup a workflow for the changes in the k8s folders. Example
https://github.com/airqo-platform/AirQo-api/blob/b3c3fe4057fd438cc7afb12544dded1697006121/.github/workflows/deploy-apis-to-staging.yml#L186

Reply:
Thanks @NoahNsimbe 
Let me set that up asap

Reply:
This has been addressed?